source/upgrade-guide/compatibility-matrix/index.rst

```diff
@@ -29,33 +29,10 @@ The Wazuh manager is also compatible with OSSEC agents but not all capabilities
 Elastic and the Wazuh Kibana plugin
 -----------------------------------
 
-The following Elastic Stack and Open Distro for Elasticsearch versions are compatible with the Wazuh manager |WAZUH_CURRENT| using the Wazuh Kibana plugin:
+Elastic Stack and Open Distro for Elasticsearch are no longer compatible with the Wazuh manager since v4.6.0.
 
-+--------------------------+---------------------------+
-| Elastic Stack version    |   Open Distro version     |
-+==========================+===========================+
-| 7.10.2                   | 1.13.2                    |
-+--------------------------+---------------------------+
-| 7.16.0–7.16.3            |                           |
-+--------------------------+---------------------------+
-| 7.17.0–7.17.11           |                           |
-+--------------------------+---------------------------+
-
-You can find more information on the `Wazuh Kibana plugin repository <https://github.com/wazuh/wazuh-kibana-app/wiki/Compatibility>`_.
-
-.. _wazuh_and_splunk_app:
 
 Splunk and the Wazuh Splunk app
 -------------------------------
 
-The following Splunk versions are compatible with the Wazuh manager |WAZUH_CURRENT| using the Wazuh Splunk app |WAZUH_SPLUNK_CURRENT|:
-
-+------------------+
-| Splunk version   |
-+==================+
-| 8.1.1–8.1.10     |
-+------------------+
-| 8.2.0–8.2.8      |
-+------------------+
-
-You can find more information on the `Wazuh Splunk app repository <https://github.com/wazuh/wazuh-splunk#installation>`__.
+Splunk is no longer compatible with the Wazuh manager since v4.6.0.
```

Comment:
```suggestion
Since v4.6.0, we no longer support Splunk. Check :doc:`/integrations-guide/index` for information on how to integrate Wazuh with it.
```